artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -2434,13 +2430,13 @@ public void run() {
                   }
                   if (++elementsIterated >= MAX_DELIVERIES_IN_LOOP) {
                      logger.debug("Breaking loop of expiring");
-                     scannerRunning.incrementAndGet();
+                     rescheduled = true;
```

Comment:
Relating to the line above this one, rather than this line...

Rather than just logging _"Breaking loop of expiring"_ it might be nice to note the queue name as the completion message does, and mention it is rescheduling.